src/wiseagents/wise_agent.py

```diff
@@ -279,7 +280,10 @@ class WiseAgentContext():
     _llm_required_tool_call : Dict[str, List[str]] = {}
     _llm_available_tools_in_chat : Dict[str, List[ChatCompletionToolParam]] = {}
     _agents_sequence : List[str] = []
-    
+    _agent_phase_assignments : Dict[str, List[List[str]]] = {}
```

Comment:
@fjuma I've added issue [#211](https://github.com/wise-agents/wise-agents/issues/211) for adding more docs here.
I've also created another more general issue to document the multi-pahse coordinator [#212](https://github.com/wise-agents/wise-agents/issues/212)

I'm moving on with this review, considering this improvements are tracked there

Reply:
@maeste Just to check, which docstrings are you referring to for https://github.com/wise-agents/wise-agents/issues/211. The variables here are internal variables so these don't have docstrings but the methods below that reference these do have docstrings.